readme.md

```diff
@@ -82,6 +89,14 @@ These instructions are only a general outline, see [the script](https://github.c
 - Update package.props
 - `dotnet restore -S directory` (where directory is replaced with the name of the directory)
 
+## Types of .NET Client Libraries to generate
+
+This code generator: autorest.csharp can generate three types of .NET client libraries:
+
+- DPG: This is Data-plane client library. Autorest.csharp will generate DPG by default. This library will only contain protocol method. Please refer to [Protocol method](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/ProtocolMethods.md). Refer to one DPG library [DeviceUpdate](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/deviceupdate/Azure.IoT.DeviceUpdate/src/Generated).
+- HLC:  This is Generation1 convenience Data-plane client library. This library will define convenience methods together with models. Refer to one HLC library [MetricsAdvisor](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Generated). Set flag `generation1-convenience-client: true` if you want to generate HLC.
```

Comment:
```suggestion
- HLC:  This is the high-level client previously generated by default.  previous generation. HLC libraries define convenience methods and models that attempt to abstract away the protocol.  Setting the flag `generation1-convenience-client: true` will generate an HLC library.
```

Reply:
> Refer to one HLC library [MetricsAdvisor](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Generated).

I would avoid calling out libraries such as this as direct examples; these may change the generator approach and invalidate the example.